verification/domain/models/individual_verification.py

```diff
@@ -0,0 +1,63 @@
+import json
+from datetime import datetime
+
+from common.utils import datetime_to_string
+from verification.constants import IndividualVerificationStatus
+from verification.domain.models.comment import Comment
+
+
+class IndividualVerification:
+    def __init__(self, verification_id, username, status, comments, created_at, updated_at):
+        self.verification_id = verification_id
+        self.username = username
+        self.status = status
+        self.comments = comments
+        self.created_at = created_at
+        self.updated_at = updated_at
+
+    @classmethod
+    def initiate(cls, verification_id, username):
+        current_time = datetime.utcnow()
+        return cls(verification_id, username, IndividualVerificationStatus.PENDING.value,
+                   [], current_time, current_time)
+
+    def approve(self):
+        self.status = IndividualVerificationStatus.APPROVED.value
+
+    def to_dict(self):
```

Comment:
You can do write like this.
```
    def to_dict(self):
        self.comments.sort(key=lambda x: x.created_at, reverse=True)
        verification_dict = {
            "verification_id": self.verification_id,
            "username": self.username,
            "status": self.status,
            "comments": [comment.to_dict() for comment in self.comments],
            "created_at": "",
            "updated_at": ""
        }
        if self.created_at is not None:
            verification_dict["created_at"] = datetime_to_string(self.created_at)
        if self.updated_at is not None:
            verification_dict["updated_at"] = datetime_to_string(self.updated_at)
        return verification_dict
```

Reply:
sorting comments logic is being used elsewhere, as well when we store dict into db we use `comment_dict_list`